bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/api/AgsHash.scala

```diff
@@ -78,7 +78,7 @@ object AgsHash {
 
     // Position Angle
     Option(ctx.getPositionAngle).foreach { a =>
-      a.degrees.## +=: buf
```

Comment:
Oh, this is because `ctx.getPositionAngle` now returns a new spModel core `Angle` instead of an old skycalc `Angle`, so the call to get the value in degrees changed accordingly.